spec/lib/vmdb/appliance_spec.rb

```diff
@@ -1,6 +1,51 @@
 require 'stringio'
 
 RSpec.describe Vmdb::Appliance do
+  describe "log_config" do
+    context "logging settings" do
+      let(:logger_io)     { StringIO.new }
+      let(:secret_filter) { [] }
+      let(:fake_settings) do
+        {
+          :authentication => {
+            :username  => "foobar",
+            :password  => "12345",  # "That's the same combination I have on my luggage!"
```

Comment:
😏

Reply:
https://c.tenor.com/l1EWDJzKD9EAAAAd/combination-luggage.gif

Reply:
Awesome, although the comment might give it away too much.  It's still not as clever as @moolitayer was with https://github.com/ManageIQ/manageiq/commit/8056603c364cea4829282558581ee6ea61de21fc#diff-b6083ae7e74136968805bfe1c555ad3ed9c6acbf406d12dd89cf99569a6d2327R4 but it's nice.

Reply:
@jrafanie now do you get credit for getting me there, or does he?  Or do you share credit?

Reply:
> @jrafanie now do you get credit for getting me there, or does he? Or do you share credit?

Never gonna tell a lie and hurt you